api/http/router.go

```diff
@@ -30,6 +30,7 @@ const (
 	DumpPath        string = versionedAPIPath + "/debug/dump"
 	BlocksPath      string = versionedAPIPath + "/blocks"
 	GraphQLPath     string = versionedAPIPath + "/graphql"
+	SchemaListPath  string = versionedAPIPath + "/schema/list"
```

Comment:
suggestion: We started changing out paths to be a bit more RESTful with the introduction of the `index` handlers. If you don't mind, please convert the 3 schema paths to a single `SchemaPath string = versionedAPIPath + "/schema"` and use the relevant http verbs in the `setRoutes` method bellow (Get, Post, Patch).

Reply:
I simplified the paths like you mentioned. Do we need to make an announcement when changing the API?

Reply:
At this stage I don't think we do as most users probably use the cli for this but the python sdk might need to be updated.